theme/debugger.less

```diff
@@ -203,6 +203,31 @@
     }
 }
 
+.debugger-preview {
+    position: fixed;
+    background-color: white;
+    color: black;
+    border: solid 1px black;
+    padding: 0.25rem;
+
+    -webkit-touch-callout: text; /* iOS Safari */
+    -webkit-user-select: text;   /* Chrome/Safari/Opera */
+    -khtml-user-select: text;    /* Konqueror */
+    -moz-user-select: text;      /* Firefox */
+    -ms-user-select: text;       /* Internet Explorer/Edge */
+    user-select: text;           /* Non-prefixed version, currently
+                                    not supported by any browser */
+    overflow-wrap: break-word;
+    max-width: 500px;
+}
+
+// When the preview appears, the text is selected so that the user can copy it.
```

Comment:
I don't think I would ever realize it's selected / copyable without the selection highlight - we do have the same issue with the share dialog as it auto selects the url and we leave the selection highlight there, maybe it's fine to leave it?

Reply:
I think it looks like "edit and continue" but broken because when you click on it the tooltip disappears. I'd rather leave it as is